src/core/scalebar/qgsnumericscalebarrenderer.cpp

```diff
@@ -33,7 +33,12 @@ QString QgsNumericScaleBarRenderer::visibleName() const
 
 int QgsNumericScaleBarRenderer::sortKey() const
 {
-  return 7;
+  return 9;
```

Comment:
I'm wondering whether it'd be worth changing it to eg. 100; then we'd have visual scale bar 0-99 and non visual bars 100-inf :)

Or we can keep increasing it as need arise, no big deal.

Reply:
Done

Reply:
Thanks.